js/src/features/project/actions.ts

```diff
@@ -0,0 +1,100 @@
+import { createAction, createAsyncAction } from "typesafe-actions"
+
+import { FormError, Paging } from "MyTypes"
+import {
+  ProjectList,
+  EditProjectStatusRequest,
+  CreateProjectRequest,
+  EditProjectDescriptionRequest,
+  EditProjectRoleRequest,
+  RemoveProjectRoleRequest,
+  CreateProjectRoleRequest,
+  Project

```

Comment:
Here should be a comma. We use prettier on this project. It puts trailing commas and semicolons. Change your IDE setting to use prettier